templates/file.j2

```diff
@@ -238,43 +254,47 @@ cards
     {%- if parts[1] -%}{{ plus() }}<span class="text-warning" title="in progress">{{parts[1]}}</span>{%- endif -%}
     {%- if parts[2] -%}{{ plus() }}<span class="text-success" title="ported">{{parts[2]}}</span>{%- endif -%}
 </small></h2>
-{{ progress.bars([data] + data.dependencies) }}
-{% set unported_deps = ([data] + data.dependencies)|rejectattr("state", "equalto", PortState.PORTED)|list%}
-{% if unported_deps %}
-    <details style="margin-bottom: 1em">
-        <summary>Show graph</summary>
-        <svg id="graph" style="overflow-x: visible; overflow-y: clip; max-height: calc(100vh - 56px); width: 100%"></svg>
-    </details>
-    <script>
-    {% set edges, nodes = data.dep_graph_data %}
-    const edges = {{edges | tojson}};
-    const nodes = {{nodes | tojson}};
-    let svg = document.getElementById('graph');
-    let has_graphed = false;
-    svg.parentElement.addEventListener('toggle', () => {
-        if (svg.parentElement.hasAttribute('open') && !has_graphed) {
-            has_graphed = true;
-            make_graph(svg, edges, nodes);
-        }
-    })
-    </script>
-    <p>The unported dependencies are</p>
-    <ul>
-        {% for d in unported_deps %}
-            <li><a href="{{ site_url }}/file/{{d.mathlib3_import|join('/')}}"><code>{{d.mathlib3_import|join('.')}}</code></a></li>
-        {% endfor %}
-    </ul>
+{% if data.dependencies is none %}
+    <div class="alert alert-danger">Dependency information not available. This file might be missing from the version of mathlib this website was built from.</div>
```

Comment:
I split this out into a separate commit that's now in master; if you merge and solve the conflicts, the diff in this section should go away.